wandb/sdk/lib/config_util.py

```diff
@@ -108,6 +109,11 @@ def dict_from_config_file(filename, must_exist=False):
         loaded = load_yaml(conf_file)
     except yaml.parser.ParserError:
         raise ConfigError("Invalid YAML in config yaml")
+    if loaded is None:
+        wandb.termwarn(
+            "Found an empty default config file. Proceeding with no defaults."
```

Comment:
Potentially identify the name of the empty file so users can get rid of this warning if they don't like seeing it